tests/test_printing.py

```diff
@@ -316,3 +318,25 @@ def random(rng, mu, size):
 
     str_repr = model.str_repr(include_params=False)
     assert str_repr == "\n".join(["x ~ CustomDistNormal", "y ~ CustomRandomNormal"])
+
+
+class TestLatexRepr:
+    @staticmethod
+    def simple_model() -> Model:
+        with Model() as simple_model:
+            error = HalfNormal("error", 0.5)
+            alpha_a = Normal("alpha_a", 0, 1)
+            Normal("y", alpha_a, error)
+        return simple_model
+
+    def test_latex_escaped_underscore(self):
```

Comment:
Instead of iterating through matches, directly assert that the LaTeX string contains only escaped underscores:
```
assert "\\_" in model_str
assert "_" not in model_str.replace("\\_", "")
```

Reply:
Yes, I agree. That is a much cleaner way of doing it. I will make the change now.